src/pages/search/advanced_search/AdvancedSearchPage.tsx

```diff
@@ -152,6 +176,19 @@ export const AdvancedSearchPage = () => {
           <StyledTypography fontWeight="bold">{t('common.language')}</StyledTypography>
           <LanguageFilter />
         </Grid>
+
+        {showFilterDivider && <StyledDivider orientation="vertical" flexItem />}
+
+        <Grid item>
```

Comment:
denne har to items i seg (`<StyledTypography>` og `<FormControlLabel>`), er det ikke da en container? 🤔 
```suggestion
        <Grid container item>
```

Reply:
Vi har ikke gjort det på de andre feltene 🤔 Må jo da begynne å definere `direction` osv, så vet ikke om det er verdt innsatsen? Med mindre du tenker at alle `children` telles som `item` uansett?

Reply:
Som diskutert: missforstelse fra meg!